addons/web/controllers/main.py

```diff
@@ -924,6 +924,21 @@ def web_login(self, redirect=None, **kw):
         response.headers['X-Frame-Options'] = 'DENY'
         return response
 
+    # sid changes once connected, therefore csrf changes too.
+    @http.route('/web/accept_scope', type='http', auth="user", methods=['GET'], website=True)
```

Comment:
Let's namespace things distinctly ;-) Also why `website=True`?

```suggestion
    @http.route('/web/auth/api/request_key', type='http', auth="user", methods=['GET'])
```

Reply:
`login_layout` breaks if website is installed and the endpoint is not marked as `website=True`.